src/evict/evict_page.c

```diff
@@ -826,15 +826,10 @@ __evict_reconcile(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags)
     else if (!WT_IS_METADATA(btree->dhandle)) {
         LF_SET(WT_REC_HS);
 
-        /*
-         * Scrub and we're supposed to or toss it in sometimes if we are in debugging mode.
-         *
-         * Note that don't scrub if checkpoint is running on the tree.
-         */
-        if (!WT_SESSION_BTREE_SYNC(session) &&
```

Comment:
@quchenhao could you check this line of the change? It reverts a change you made in WT-9751, but I can't see any context about why it matters.

Reply:
I can't remember much. I think the reason is that checkpoint itself can do eviction. This is to prevent checkpoint eviction to keep the content in memory and increase the workload for checkpoint.